include/MiniFB_ex.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "MiniFB_ex_enums.h"
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+    int mfb_open_ex(const char* name, int width, int height, int flags);
+    eBool mfb_set_viewport(unsigned offset_x, unsigned offset_y, unsigned width, unsigned height);
+
+    typedef void(*mfb_active_func)(eBool isActive);

```

Comment:
I think it would be nicer that `mfb_open_ex` would return a handle to a Window instead. I have read your mail yet but that would allow multiple windows also if needed

Reply:
I would prefer to do this change (multiple windows) in the next pull request